### ISSUE TITLE
Get rid of X{} references in docstrings

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#3415]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)| @thijstriemstra reported|
|-|-|
|Trac ID|trac#3415|
|Type|defect|
|Created|2008-09-06 23:41:58Z|
|Branch|https://github.com/twisted/twisted/tree/x-epytext-3415|

There are couple of files that use X{} instead of C{}, see attached patch that fixes this problem.

Attachments:

* [epydoc-x-error-3415.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/59f/59f1d794faeb57b7b54da51760c2d9207536cfa6/f72e7e15783924d449fa3d661357fd3c512a62c1.patch) (5909 bytes) - added by thijs on 2008-09-06 23:42:53Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__3415 3415
type__defect defect
reporter__thijs thijs
priority__normal normal
milestone__ 
branch__branches_x_epytext_3415 branches/x-epytext-3415
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1220744518000000 1220744518000000
changetime__1220911663000000 1220911663000000
version__None None
owner__ 
cc__thijs
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)|@thijstriemstra removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_3"></a>|@exarkun set owner to @thijstriemstra|
|-|-|

Looks pretty good.  I'd suggest putting it into a branch and testing it on the documentation builder.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_4"></a>|@thijstriemstra commented|
|-|-|

(In [24725]) Branching to 'x-epytext-3415'

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_5"></a>|@thijstriemstra removed owner|
|-|-|

Replying to [exarkun](#note_3):
> Looks pretty good.  I'd suggest putting it into a branch and testing it on the documentation builder.

Doc builder [didn't](http://buildbot.twistedmatrix.com/builders/documentation/builds/800) complain, putting the branch up for review.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_6"></a>|@exarkun set owner to @thijstriemstra|
|-|-|

Great, please merge.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_7"></a>|@thijstriemstra set status to `closed`|
|-|-|

(In [24746]) Merge x-epytext-3415: Get rid of X{} references in docstrings, replacing them with C{}.

Author: thijs
Reviewer: exarkun
Fixes: [#3415](https://github.com/twisted/twisted/issues/3415)

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
